src/algebra/big_operators/basic.lean

```diff
@@ -1026,6 +1035,12 @@ lemma sum_range_succ' [add_comm_monoid β] (f : ℕ → β) :
 @prod_range_succ' (multiplicative β) _ _
 attribute [to_additive] prod_range_succ'
 
+lemma sum_range_add {β} [add_comm_monoid β] (f : ℕ → β) (n : ℕ) (m : ℕ) :
+  (∑ x in range (n + m), f x) =
+  (∑ x in range n, f x) + (∑ x in range m, f (n + x)) :=
+@prod_range_add (multiplicative β) _ _ _ _
+attribute [to_additive] prod_range_add
```

Comment:
I assume it doesn't work if you just put this on line 637?

Reply:
I wanted sum_range_add immediately after sum_range_succ' and prod_range_add immediately after prod_range_succ' because of how I view the relationship between the lemmas. Is that a problem?

Reply:
Also, moving sum_range_add to line 646 works though not line 637. I tested it just now.